asv/plugins/virtualenv.py

```diff
@@ -108,24 +97,30 @@ def name(self):
 
     @classmethod
     def matches(self, python):
-        if not (re.match(r'^[0-9].*$', python) or re.match(r'^pypy[0-9.]*$', python)):
+        match = re.match(r'\A(?:pypy)?([0-9.]*)\Z', python)
+        if not match:
             # The python name should be a version number, or pypy+number
             return False
+        executable = Virtualenv._find_python(python)
 
-        try:
-            import virtualenv
-        except ImportError:
-            return False
+        if LooseVersion(match.group(1)) < LooseVersion("3.3"):
+            self._virtualenv_argv = [
+                executable, "-mvirtualenv", "--no-site-packages"]
```

Comment:
Maybe it should use current python, when using virtualenv?
venv is bundled, but Virtualenv is not necessarily installed in the target Pythons...